byron/ledger/executable-spec/src/Ledger/Delegation.hs

```diff
@@ -387,8 +432,8 @@ instance STS SDELEGS where
         case sig of
           [] -> return st
           (x:xs) -> do
-            dss' <- trans @SDELEGS $ TRC (env, st, xs)
-            dss'' <- trans @SDELEG $ TRC (env, dss', x)
+            dss'  <- trans @SDELEG $ TRC (env, st, x)
```

Comment:
I'd argue this is the reverse order of applying signals than how it's stated in the LaTeX ledger spec. For `Γ;c` where `c` is the last signal in the sequence, `c` gets applied on top of the state obtained by applying `Γ`. Isn't that right?

Reply:
@mdimjasevic As you said, `c` get's applied on top of `Gamma`, so that's a left to right traversal. The pattern match is different, because the spec uses snoc lists, but the traversal direction is correct, i.e. head first.

Reply:
So neither the old implementation nor the suggested fix are correct, right?

Reply:
The suggest fix is correct - just focus on the order of which certs get processed

Reply:
@ruhatch, you're right!

@dnadales, my apologies for requesting changes here, I was wrong.